statics/js/components/capture-form.js

```diff
@@ -80,6 +80,16 @@ Vue.component('capture-form', {
               <label for="netflowTarget">NetFlow collector address, if empty the agent will be used</label>\
               <input id="netflowTarget" type="text" class="form-control input-sm" v-model="target"/>\
             </div>\
+            <div class="form-group" v-if="captureType != \'ovssflow\' && captureType != \'ovsnetflow\'">\
+              <label for="target">Target</label>\
```

Comment:
It possible to specify a target for ovssflow and ovsnetflow but not the target type as we know if it is sflow or netflow. So the Target field should be outside of the if statement